scripts/tasks/statistics

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Run a Fledge south service written in Python
+# Run a Fledge south service written in C
```

Comment:
Not a Fledge south service but Fledge task

Reply:
fixed